tests/services/Schematic_PluginsServiceTest.php

```diff
@@ -38,7 +38,6 @@ public function setUp()
     public static function setUpBeforeClass()
     {
         parent::setUpBeforeClass();
-        require_once __DIR__.'/../../SchematicPlugin.php';
         require_once __DIR__.'/../../models/Schematic_ResultModel.php';
```

Comment:
That loader only works in scripts run by the schematic bootstrapper. That doesn't include tests.